site/viamguides/src/plant-watering-device-workshop/plant-watering-device-workshop.md

```diff
@@ -0,0 +1,182 @@
+author: Hazal Mestci
+id: plant-watering-device-workshop
+summary: Build a Plant Watering Device
+categories: Getting-Started, Developer
+environments: web
+status: Draft
+feedback link: <https://github.com/viam-labs/viamcodelabs/issues>
+tags: Getting Started, Developer, Hardware
+
+# Plant Watering Device Workshop
+<!-- ------------------------ -->
+## Overview
+
+Duration: 5
+
+Building a useful robot doesn’t have to require complicated code or expensive equipment. With a Raspberry Pi and some cheap, basic hardware, you can keep your plants healthy and happy from anywhere in the world!
+
+Follow this workshop to learn how to set up an automatic plant watering system by wiring the components of the device, configuring it in the Viam app, and coding a simple script to activate the water pump motor.
+
+### Prerequisites
+
+- Sign up for a free Viam account, and then [sign in](https://app.viam.com/robots/) to the Viam app.
+- Follow the Raspberry Pi [setup guide](https://docs.viam.com/installation/prepare/rpi-setup/) and flash a Viam-compatible operating system on the Pi. Make sure you are able to SSS into it.
+- [Purchase](https://www.amazon.com/hz/wishlist/ls/3D7AOY2LX3PH2) or acquire the hardware necessary for the workshop: a Raspberry Pi with micro-sd card and a power supply, a webcam (optional for the additional sections), a 5V water pump with tubing, a relay module, a moisture sensor, bunch of jumper wires, and wire connectors.
+- Have a planter box or flower pot, a screw driver and a water container ready.
+
+### What You’ll Learn
+
+- How to physically assemble a functional plant watering device by connecting and wiring the components.
+- How to configure the device’s components using the Viam app.
+- How to write Python code using the Viam Python SDK to control the plant watering device by using custom sensor and motor methods.
+- How to test and adjust the system for optimal performance.
+
+### What You’ll Need
+
+- All the hardware components listed in prerequisites.
+- Python and necessary libraries.
+- A [GitHub](https://github.com/) account.
+- [VSCode](https://code.visualstudio.com/download) installed, or another similar code editor of your choice.
+
+### What You'll Build
```

Comment:
I think this should go above the `What You'll Learn` heading to add context to what readers will learn when building this project.